scripts/create-client-config.sh

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env bash
+#
+# k8ssandra-operator should be installed in the remote clusters prior to running this. The
+# script fetches the k8ssandra-operator service account from the remote cluster and
+# extracts the token and CA cert which are then added to a kubeconfig file. The script then
+# creates a secret with the contents of the kubeconfig file. Lastly, the script creates a
+# ClientObject that references the secret.
+#
+# This script requires the following to be installed:
+#
+#    - kubectl
+#    - yq
+#
+# TODO Accept multiple values for the src-context option and generate a kubeconfig with
+#      entries for each
+
+set -e
+
+OPTS=$(getopt -o h --long src-context:,src-kubeconfig:,dest-context:,dest-kubeconfig:,namespace:,serviceaccount,output-dir:,help -n 'create-client-config' -- "$@")
+
+eval set -- "$OPTS"
+
+function help() {
+    echo
+    echo "Syntax: create-client-config.sh [options]"
+    echo "Options:"
+    echo "src-context     The context for the source cluster that contains the service account. This or the src-kubeconfig option must be set."
```

Comment:
Very minor: for a prettier output you could align the start of each description text.

Reply:
I tried a couple different approaches for formatting/wrapping the text into fixed width columns. Any suggestions on how best to do it?

Reply:
I'm no specialist in these things but I added a suggestion in another comment below:

https://github.com/k8ssandra/k8ssandra-operator/pull/89/files#r693825451